pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -318,7 +317,17 @@ func outputKubeadmInitSteps(logs io.Reader, wg *sync.WaitGroup) {
 			continue
 		}
 		register.Reg.SetStep(nextStep.registerStep)
-		out.Step(style.SubStep, nextStep.stepMessage)
+		// because the translation extract (make extract) needs simple strings to be included in translations we have to pass simple strings
+		if nextStepIndex == 0 {
+			out.Step(style.SubStep, "Generating certificates and keys ...")
+		}
+		if nextStepIndex == 1 {
+			out.Step(style.SubStep, "Booting up control plane ...")
+		}
+		if nextStepIndex == 2 {
+			out.Step(style.SubStep, "Configuring RBAC rules ...")
```

Comment:
Would it make more sense to just set a variable in the `if` ladder and do that `out.Step` all outside. So if we change something don't have to change it three times.

```
var msg string
if nextStepIndex == 0 {
  msg = "Generating certificates and keys ..."
}
if nextStepIndex == 1 {
  msg = Booting up control plane ..."
}
if nextStepIndex == 2 {
  msg = "Configuring RBAC rules ..."
}
out.Step(style.SubStep, msg)
```

Reply:
And maybe move this into it's own function that accepts `nextStepIndex` as a param?

Reply:
aaah just tried that the `make extract` wasn't smart enough to catch it

Reply:
Ah, I see